aws_lambda_powertools/utilities/data_classes/cloudformation_custom_resource_event.py

```diff
@@ -0,0 +1,52 @@
+from enum import Enum
+from typing import Any, Dict, Optional
+
+from aws_lambda_powertools.utilities.data_classes.common import DictWrapper
+
+
+class CloudFormationRequestType(Enum):
+    CREATE = "Create"
+    UPDATE = "Update"
+    DELETE = "Delete"
```

Comment:
These values are static and only used once in the class, I think it makes more sense to remove this `Enum` and add it as a `Literal` in the `request_type` property.